CHANGLOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on Keep a Changelog,
+and this project adheres to Semantic Versioning.
+
+## [0.0.1] - 2025-03-11
+
+### Added
+
+Implemented /version endpoint to return application version from pom.xml.
+
+Created CHANGELOG.md to track project updates.
+
+Configured maven-jar-plugin to include pom.properties file in build artifacts.
```

Comment:
Please write this together with: `Implemented /version endpoint to return application version from pom.xml.`. This is one feature that you implemented